static/js/customers.js

```diff
@@ -2352,7 +2352,7 @@ if ( document.body.className.match(new RegExp('(\\s|^)customers(\\s|$)')) !== nu
 
 			if( $quantity.val() &&
 				$quantity.val().length ) {
-				$quantity.val(parseInt($quantity.val()));
+				$quantity.val(Math.round($quantity.val()*1000) / 1000);
```

Comment:
This line should still have parseFloat in it technically - though Javascript will usually handle it correctly, it's always good to just tell it what to do when it comes to math.

```
$quantity.val(Math.round(parseFloat($quantity.val())*1000) / 1000);
```